src/game/CowGame.tsx

```diff
@@ -28,13 +29,23 @@ export function CowGame() {
 
   // Hide box when pressing Space
   const hideMessageBox = () => setShowMessageBox(false);
-  useKeyPress(["Space", "ArrowUp"], hideMessageBox);
+
+  // Desktop/non-touch device: Key binding to hide message box
+  !!IS_MOBILE && useKeyPress(["Space", "ArrowUp"], hideMessageBox);
```

Comment:
Non-touch device? useKeyPress listeners.

Reply:
In that case, there's an extra `!`

Reply:
Yep, testing it locally, only works on desktop if I remove the extra `!`.
On mobile (and desktop with small screen size) the message never goes away.

Reply:
Whoops, you're right. I wrongfully opted to use the 'not not' operator here. Thanks for finding it.

Reply:
`!!` = `Boolean(...)`